test/alternator/test_metrics.py

```diff
@@ -191,6 +191,14 @@ def test_wcu(test_table_s, metrics):
         val2 = 'a' * (2 * KB - total_length + 1) # message length 2K + 1
         test_table_s.put_item(Item={'p': p, 'att': val, 'another': val2})
 
+def test_wcu_batch_write_items(test_table_s, metrics):
+    with check_increases_operation(metrics, ['PutItem'], 'scylla_alternator_wcu_total', 6):
```

Comment:
I'm confused - why does this increase by 6, not by 3? It seems below you have one tiny write plus one larger one that takes 2. Isn't this 3 WCUs?

Reply:
yes, but we are counting half units, like we do for rcu

Reply:
This is very confusing and unintuitive. Where did we document this?
Please correct me if I'm wrong, but aren't all metrics in Scylla floating points, not integers? If so, why don't we use 0.5 for half units?

Reply:
The RCU and wcu are for our internal use, I preferred keeping it with round numbers; we may change to switch to unsigned in the future

Reply:
These are user-visible metrics.

But no matter if it's for end-users or for Scylla developers, I think it's just confusing that you see in a graph "10000 WCU" and need to mentally convert it into "really, it's 5000 WCU". What is this confusion good for? Why can't we just write 5000? I'm 100% sure that a year after you release this, NOBODY, including Scylla developers (except possibly Avi), will remember the numbers you see in the graph need to be halved before used. The name of the metric "scylla_alternator_wcu_total" suggests that this is "wcu", not some weird function of wcu.